app/docs/src/reference/attribute.md

```diff
@@ -0,0 +1,87 @@
+# Attribute Functions
+
+Attribute functions are a kind of [function](./function.md) used to set
+properties on [components](./components.md) using data from other properties on
+the same or other components. They are the primary mechanism for transforming
+data within a component.
```

Comment:
They are not the primary method of transforming data - transformation funcs are :D